Core.DomainModel/Organization/OrganizationUnit.cs

```diff
@@ -150,6 +150,26 @@ public void ConvertToKitosUnit()
             ExternalOriginUuid = null;
         }
 
+        private bool CanBeModified()
+        {
+            return HasParent() && IsOfKitosOrigin();
+        }
+
+        private bool CanChangeName()
```

Comment:
Remember that the purpose is not only to provide info for the UI but to make sure that these rules are obeyed by the mutation methods... as far as I can see they are not.

Since we have "system" changes going on, it must be possible to "override" when being called from the strategy, so the different "ChangeName...relocate etc should have an optional enforce parameter used by the strategy"